### DISCUSSION TITLE
Robotik Programlama | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#116] [Site]
# Robotik Programlama | SonsuzUs Github

Robotik kodlama nedir?

https://sonsuzus.github.io/posts/robotik/

**Comment by sonsuzus** [comment]
Robotik kodlama fazlasıyla abartılan bir konu ayrıca. Öncelikle öğrencilerin ve çocukların biraz yazılım bilgisi ve algoritma öğrenmeleri gerekiyor. Daha çok konuya hakim olurlar böylece.
